libs/genai/langchain_google_genai/llms.py

```diff
@@ -4,6 +4,7 @@
 from typing import Any, Callable, Dict, Iterator, List, Optional, Union
 
 import google.api_core
+import google.cloud.aiplatform
```

Comment:
why do you need this import? the genai package doesn't have the dependency on aiplatform.

Reply:
My mistake. I remove this import and fixed formatting on embeddings file.